site/content/open-guide-to-kanban/2025.7/index.pl.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Open Guide to Kanban – w kontekście pracy umysłowej
+short_title: Open Guide to Kanban
+description: Open Guide to Kanban stanowi bezpłatny, oparty na społeczności przewodnik referencyjny stosowania Kanban w pracy umysłowej. Definiuje podstawowe praktyki oraz metryki niezbędne do poprawy przepływu (flow), optymalizacji dostarczania wartości i wzmocnienia trwałości zespołu. Niniejszy przewodnik wspiera skalowalne wdrożenia systemu Kanban w różnych branżach oraz uzupełnia inne podejścia agile, lean i oparte na przepływie.
+keywords:
+  - Kanban
+  - Open Guide to Kanban
+  - knowledge work
+  - flow optimisation
+  - WIP limits
+  - value delivery
+  - agile
+  - lean
+  - continuous improvement
+  - service level expectation
+  - cumulative flow
+  - throughput
+  - metrics
+  - work item age
+  - flow efficiency
+  - visualisation
+  - work in progress
+  - process improvement
+  - kanban board
+  - definition of workflow
+  - outcome-oriented delivery
+author:
+  - John Coleman
+date: 2025-07-02T09:00:00Z
+type: guide
+forked_from: the-kanban-guide/2025.5
+lang: pl
+mainfont: "Times New Roman"
+sansfont: "Arial"
+monofont: "Courier New"
+sitemap:
+  priority: 1.0
+aliases:
+  - /open-guide-to-kanban/latest/
```

Comment:
All aliass need to be prefixed with the language. Can you add `/pl/` to the front of them all